core/test/log/convergence.cpp

```diff
@@ -113,12 +123,13 @@ TYPED_TEST(Convergence, CanLogData)
 TYPED_TEST(Convergence, DoesNotLogIfNotStopped)
 {
     auto logger = gko::log::Convergence<TypeParam>::create(
-        gko::log::Logger::criterion_events_mask);
+        gko::log::Logger::criterion_events_mask |
+        gko::log::Logger::iteration_complete_mask);
```

Comment:
We should maybe make these the default of `gko::log::Convergence::create()` actually.

Reply:
I changed the constructor of `log::Convergence` as you suggested